oci/justfile

```diff
@@ -0,0 +1,32 @@
+set shell := ["bash", "-cu"]
+
+@default:
+    echo "Available Commands:"
+    echo ""
+    echo "  tests *args"
+    echo "    Runs the test suite using pytest with the specified parameters."
+    echo "    Usage:"
+    echo "      tests [pytest-params]"
+    echo "    Example:"
+    echo "      tests -v --maxfail=2"
+    echo ""
+    echo "  report *args"
+    echo "    Generates reports using the run.sh script."
+    echo "    Usage:"
+    echo "      report <Test_Category>"
+    echo "    Valid Test Categories:"
+    echo "      full, versioning, basic, policy, cold, locking, big-objects"
+    echo "    Example:"
+    echo "      report full"
+    echo ""
```

Comment:
nao precisa de help com echo, pode só usar o just list built-in, para documentar cada recipe e estas descriptions aparecerem no list, basta um comment em cima de cada recipe se nao me engano